qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -14,12 +14,45 @@
 from qiita_core.util import execute_as_transaction
 from qiita_pet.handlers.api_proxy.util import check_access, check_fp
 from qiita_db.metadata_template.util import load_template_to_dataframe
-from qiita_db.util import convert_to_id
+from qiita_db.util import convert_to_id, get_files_from_uploads_folders
 from qiita_db.study import Study
 from qiita_db.ontology import Ontology
 from qiita_db.metadata_template.prep_template import PrepTemplate
 
 
+def new_prep_template_get_req(study_id):
+    """
+    Parameters
+    ----------
+    study_id : int
+        The study id
+
+    Returns
+    -------
+    (list of str, list of str, dict of {str: list of str})
+        The list of txt,csv files in the upload dir for the given study
```

Comment:
I think this is actually a typo in the documentation. We are supporting tsv (Tab Separated Values) rather than csv (Comma Separated Values).